src/Sylius/Bundle/ApiBundle/Command/Admin/SendResetPasswordEmail.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Admin;
+
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
+use Sylius\Bundle\ApiBundle\Command\LocaleCodeAwareInterface;
+
+class SendResetPasswordEmail implements LocaleCodeAwareInterface, IriToIdentifierConversionAwareInterface
```

Comment:
Same here

Reply:
I've unresolved Mati comment, as the line didn't change and he was pointing to some issues. @Zales0123 what was it?

Reply:
@lchrusciel it was about missing experimental.

Reply:
And I've added it, GitHub just doesn't show this as outdated.

Reply:
Indeed, the comment is fixed 👍